features/account/customer_account/address_book/viewing_addresses.feature

```diff
@@ -0,0 +1,23 @@
+@customer_account
+Feature: Viewing customer's addresses
+    In order to see all my addresses in address book
+    As a Customer
+    I want to be able to browse my addresses
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is a customer "John Doe" identified by an email "doe@example.com" and a password "banana"
+        And this customer has an address "John Doe", "Banana Street", "90232", "New York", "United States" in address book
```

Comment:
Yups, beacause we want to be sure, that you will see only you addresess on the list. In scenario about viewing addresses I can add an step "And I should not see address assigned to "John Doe".